packages/appium/docs/en/sponsors.md

```diff
@@ -0,0 +1,54 @@
+---
+hide:
+  - navigation
+  - toc
+
+title: Sponsors & Backers
+---
```

Comment:
Suggest adding some CSS to hide the sponsors footer on this page, like on the main page:
```
<style>
  .appium-sponsor-thanks {
    display: none;
  }
</style>
```